### ISSUE TITLE
Grid Search docs error

### ISSUE THREAD

**Issue by namebrandon** [aigamedev/scikit-neuralnetwork#75]
quotation/tick marks are needed around learning_rate (and I assume any other parameter to be tested). The example in the docs throws an error due to learning_rate not being enclosed by quotation marks.

    gs = GridSearchCV(nn, param_grid={
        learning_rate: [0.05, 0.01, 0.005, 0.001],
        'hidden0__units': [4, 8, 12],
        'hidden0__type': ["Rectifier", "Sigmoid", "Tanh"]})
    gs.fit(a_in, a_out)

**Comment by namebrandon** [comment]
pull req submitted - https://github.com/aigamedev/scikit-neuralnetwork/pull/76

**Comment by alexjc** [comment]
Closed by #79.
